DECISION-MAKING.md

```diff
@@ -0,0 +1,79 @@
+**Contributor** - a person listed in the Appendix I: Contributors. The 
+community decides on the content of this file using the same process as any 
+other change to the Repository (see below) allowing the meaning of "Contributor"
+to evolve independently of the Decision-making rules.
+
+## Rules
+1. Every modification of the specification (including a correction of a typo, 
+   adding a new Contributor, an extension adding support for a new data type, or
+   others) or proposal to release a new version needs to be done via a Pull 
+   Request (PR) to the Repository.
+1. Anyone can open a PR (this action is not limited to Contributors).
+1. A PR is eligible to be merged if and only if these conditions are met:
+   1. The last commit is at least 5 working days old to allow the community to 
+      evaluate it.
+   1. The PR features at least one [Review that Approves](https://help.github.com/articles/about-pull-request-reviews/#about-pull-request-reviews) 
+      the PR from a Contributor that is not an author of the PR. The review 
+      needs to be made after the last commit in the PR (equivalent to 
+      [Stale review dismissal](https://help.github.com/articles/enabling-required-reviews-for-pull-requests/)
+      option on GitHub).
+   1. Does not feature any [Reviews that Request changes](https://help.github.com/articles/about-required-reviews-for-pull-requests/).
+   1. Does not feature "WIP" in the title (Work in Progress).
+   1. Passes all automated tests.
+1. Any Contributor can Review a PR and Request changes. If a Contributor 
+   Request changes they need to provide an explanation what changes 
+   should be added and justification of their importance. Reviews requesting 
+   changes can also we used to request more time to review a PR.
+1. A Contributor that Requested changes can Dismiss their own review or Approve 
+   changes added by the Contributor who opened the PR.
+1. If the author of a PR and Contributor who provided Review that Requests 
+   changes cannot find a solution that would lead to the Contributor dismissing 
+   their review or accepting the changes the Review can be Dismissed with a 
+   vote. Rules governing voting:
+   1. A Vote can be triggered by any Contributor, but only after 5 working days 
+      from the time a Review Requesting Changes has been raised and in case a 
+      Vote has been triggered previously no sooner than 15 working days since 
+      its conclusion.
+   1. Only Contributors can vote, each contributor gets one vote.
+   1. A Vote ends after 5 working days or when all Contributors have voted 
+      (whichever comes first).
+   1. A Vote freezes the PR - no new commits or Reviews Requesting changes can 
+      be added to it while a vote is ongoing. If a commit is accidentally made 
+      during that period it should be reverted.
+   1. The quorum for a Vote is 30% of all Contributors.
```

Comment:
All contributors over time, or just for the PR?

If it's over time, this may become infeasible as people move on in their careers.

Reply:
All contributors. Your concern is indeed valid, but over time we can adjust this percentage. I previously considered an option for a quorum consisting of recent commit authors, but that did not count other contributors to the BIDS ecosystem, so I left it out.